src/test/scala/reporter/CloudWatchReporterSpec.scala

```diff
@@ -0,0 +1,35 @@
+package reporter
+
+import com.blacklocus.metrics.CloudWatchReporterBuilder
+import com.codahale.metrics.{Metric, MetricFilter, MetricRegistry}
+import org.scalamock.scalatest.MockFactory
+import org.scalatest.{FlatSpec, Matchers, PrivateMethodTester}
+import Utils.buildMetricFilter
+import scala.util.matching.Regex
+
+
+class CloudWatchReporterSpec extends FlatSpec with Matchers with PrivateMethodTester with MockFactory {
+
+  private val metricRegistry: MetricRegistry = new MetricRegistry
+  private val metric: Metric = mock[Metric]
+
+
+  "Metrics filter" should "match any metric when empty string filter is given" in {
```

Comment:
This should be `buildMetricFilter` instead of `Metrics filter` to make it easier to cross reference